migration/src/m0000300_create_product_version.rs

```diff
@@ -55,7 +55,8 @@ impl MigrationTrait for Migration {
                     .table(ProductVersion::Table)
                     .name(INDEX_BY_PID_V)
                     .if_not_exists()
-                    .unique()
+                    //TODO what about duplicate version for the same product?
```

Comment:
I suppose if duplicate, just use the existing row?  upsert behaviour, instead of duplicates?

Reply:
Yeah, this uniqueness works fine in manual testing, but it makes some tests failure. It was just left here to investigate these failures further and see if it is a valid use case to have these duplicate values.